examples/chip-tool-darwin/commands/common/CHIPCommandBridge.mm

```diff
@@ -91,23 +83,15 @@ static void OnResponseTimeout(chip::System::Layer *, void * appState)
         }
     }
     LogErrorOnFailure(chip::DeviceLayer::PlatformMgr().StopEventLoopTask());
```

Comment:
Likewise followup: we should not have manual `StopEventLoopTask` calls.  We should shut down the controller instead.

Reply:
Ok. Created a follow up [here](https://github.com/project-chip/connectedhomeip/issues/17365).